airbyte-integrations/connectors/source-gitlab/metadata.yaml

```diff
@@ -10,7 +10,7 @@ data:
   connectorSubtype: api
   connectorType: source
   definitionId: 5e6175e5-68e1-4c17-bff9-56103bbb0d80
-  dockerImageTag: 2.1.1
+  dockerImageTag: 2.2.0
```

Comment:
Should this be the major version 3.0.0, as the schema for stream `merge_request_commits` changed?